static/js/functions.js

```diff
@@ -87,8 +90,10 @@ $(document).ready(function () {
             { data : "number" },
             { data : "title" },
             { data : "sender"},
-            { data : "updated_at"},
-            { data : "hours"},
+            {
+                data : "updated_at",
+                render: convertDate
```

Comment:
Don't you need to provide data and type parameters when calling `convertDate` method?